app/Admin/Controllers/Management/AccessController.php

```diff
@@ -62,9 +62,29 @@ public function index(Content $content)
                     $query->where('parent_id', '=', $this->input);
                 });
             }, 'Pôle')->select($poles);
+
+            $filter->where(function ($query) {
+                $query->whereHas('member', function ($query) {
+                    $query->whereRaw('lower(lastname) like ?', [strtolower("%{$this->input}%")]);
+                });
+            }, 'Nom du membre');
+            $filter->where(function ($query) {
+                $query->whereHas('member', function ($query) {
+                    $query->whereRaw('lower(firstname) like ?', [strtolower("%{$this->input}%")]);
+                });
+            }, 'Prénom du membre');
+
+            $filter->where(function ($query) {
+                $query->whereHas('semester', function ($query) {
+                    $query->where('id', '=', $this->input);
+                });
+            }, 'Semestre')->select(Semester::get(['id', 'name'])->pluck('name', 'id'));
         });
 
-        $grid->get()->pôle()->sortable()->display(function () {
+        // Add all columns.
+        $grid->column('asso.shortname', 'Asso');
+
+        $grid->get()->pôle()->display(function () {
```

Comment:
Didn't see it last time but I'm not a huge fan of accents in variable and function names. It there a way to avoid it? Maybe with an argument.